index.html

```diff
@@ -1174,7 +1177,15 @@ <h2>The <dfn>ConsumedThing</dfn> interface</h2>
         Promise&lt;undefined&gt; stop(optional InteractionOptions options = null);
       };
 
-      typedef object PropertyMap;
+      [SecureContext, Exposed=(Window,Worker)]
+      interface PropertyReadMap {
+        readonly maplike&lt;DOMString, InteractionOutput&gt;;
+      };
+
+      [SecureContext, Exposed=(Window,Worker)]
+      interface PropertyWriteMap {
+        readonly maplike&lt;DOMString, InteractionInput&gt;;
+      };
```

Comment:
The WebIDL syntax does not seem very intuitive to me. I would have expected something like

`interface PropertyReadMap extends Map<DOMString, InteractionOutput>`.

Anyhow, `WebIDL maplikes` are to be described like this in the body.

Even though I raised the issue I am not sure myself whether if it is worth using maplikes ...

Reply:
Web IDL syntax. https://heycam.github.io/webidl/#idl-maplike

Reply:
This is the way to express the idea in Web IDL, still better than using `object`.